src/main/kotlin/net/ltgt/gradle/errorprone/ErrorPronePlugin.kt

```diff
@@ -201,7 +201,8 @@ internal class ErrorProneCompilerArgumentProvider(
 
     override fun asArguments(): Iterable<String> {
         return when {
-            errorproneOptions.isEnabled.getOrElse(false) -> listOf("-Xplugin:ErrorProne $errorproneOptions", "-XDcompilePolicy=simple")
+            // should-stop.ifError is for JDK 9+, shouldStopPolicyIfError for JDK 8; it's safe to indiscriminately pass both
+            errorproneOptions.isEnabled.getOrElse(false) -> listOf("-Xplugin:ErrorProne $errorproneOptions", "-XDcompilePolicy=simple", "-XDshould-stop.ifError=FLOW", "-XDshouldStopPolicyIfError=FLOW")
```

Comment:
This all LGTM.

Passing the preferred form of the flag depending on the current JDK version feels slightly cleaner to me, but I don't see any real downsides to always passing both flags and using the `-XD` form of the flag.

Reply:
Thanks for the review Liam :+1: 

My intent is to cut a release this week end, and I can later iterate to improve things. I'll see if I can implement the JDK-specific flag variant since then or not. At some point I'll likely drop support for older JDKs as well, so I'm still pondering whether this would be time well spent, or just wasted as it'll eventually be replaced by a hard-coded `--should-stop=ifError=FLOW`.